website/src/components/SearchPage/DownloadDialog/LinkOutMenu.tsx

```diff
@@ -189,7 +189,7 @@ function LinkOutMenuDataUseTermModal(props: {
                         className='px-4 py-2 bg-primary-600 text-white rounded-md font-medium hover:bg-primary-700 transition-colors'
                         onClick={props.onClick1}
                     >
-                        Include Restricted-Use
+                        Include restricted-use
```

Comment:
In Pathoplexus DUT (https://pathoplexus.org/about/terms-of-use/data-use-terms), we write for example "governing how Restricted-Use Data can be used and shared onward, as well as how it must be attributed when used", i.e., specifically capitalizing Restricted-Use. Should we do it here as well?